ignite/templates/app/files-consumer/app/app.go.plush

```diff
@@ -0,0 +1,460 @@
+	AccountKeeper         authkeeper.AccountKeeper
+	BankKeeper            bankkeeper.Keeper
+	ConsensusParamsKeeper consensuskeeper.Keeper
+	SlashingKeeper        slashingkeeper.Keeper
+	ConsumerKeeper        ibcconsumerkeeper.Keeper
+	CrisisKeeper          *crisiskeeper.Keeper
+	UpgradeKeeper         *upgradekeeper.Keeper
+	ParamsKeeper          paramskeeper.Keeper
+	AuthzKeeper           authzkeeper.Keeper
+	EvidenceKeeper        evidencekeeper.Keeper
+	FeeGrantKeeper        feegrantkeeper.Keeper
+	GroupKeeper           groupkeeper.Keeper
+	NFTKeeper             nftkeeper.Keeper
+	CircuitBreakerKeeper  circuitkeeper.Keeper
+
+    // IBC
+	IBCKeeper           *ibckeeper.Keeper // IBC Keeper must be a pointer in the app, so we can SetRouter on it correctly
+	CapabilityKeeper    *capabilitykeeper.Keeper
+	IBCFeeKeeper        ibcfeekeeper.Keeper
+	ICAControllerKeeper icacontrollerkeeper.Keeper
+	ICAHostKeeper       icahostkeeper.Keeper
+	TransferKeeper      ibctransferkeeper.Keeper
+
+	// Scoped IBC
+	ScopedIBCKeeper           capabilitykeeper.ScopedKeeper
+	ScopedIBCTransferKeeper   capabilitykeeper.ScopedKeeper
+	ScopedICAControllerKeeper capabilitykeeper.ScopedKeeper
+	ScopedICAHostKeeper       capabilitykeeper.ScopedKeeper
+
+	// this line is used by starport scaffolding # stargate/app/keeperDeclaration
+
+	// simulation manager
+	sm *module.SimulationManager
+}
+
+func init() {
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		panic(err)
+	}
+
+	DefaultNodeHome = filepath.Join(userHomeDir, "."+Name)
+}
+
+// AppConfig returns the default app config.
+func AppConfig() depinject.Config {
+	return depinject.Configs(
+		appConfig,
+		// Loads the app config from a YAML file.
+		// appconfig.LoadYAML(AppConfigYAML),
+		depinject.Supply(
+			// supply custom module basics
+			map[string]module.AppModuleBasic{
+				genutiltypes.ModuleName: genutil.NewAppModuleBasic(genutiltypes.DefaultMessageValidator),
+				// this line is used by starport scaffolding # stargate/appConfig/moduleBasic
+			},
+		),
+	)
+}
+
+// New returns a reference to an initialized App.
+func New(
+	logger log.Logger,
+	db dbm.DB,
+	traceStore io.Writer,
+	loadLatest bool,
+	appOpts servertypes.AppOptions,
+	baseAppOptions ...func(*baseapp.BaseApp),
+) (*App, error) {
+	var (
+		app        = &App{}
+		appBuilder *runtime.AppBuilder
+
+		// merge the AppConfig and other configuration in one config
+		appConfig = depinject.Configs(
+			AppConfig(),
+			depinject.Supply(
+				// Supply the application options
+				appOpts,
+        // Supply with IBC keeper getter for the IBC modules with App Wiring.
+        // The IBC Keeper cannot be passed because it has not been initiated yet.
+        // Passing the getter, the app IBC Keeper will always be accessible.
+        // This needs to be removed after IBC supports App Wiring.
+        app.GetIBCKeeper,
+        app.GetCapabilityScopedKeeper,
+				// Supply the consumer keeper for the consumer module
+				&app.ConsumerKeeper,
```

Comment:
I think the `ConsumerKeeper` is not initialized yet to be injected here

Reply:
Let's fix the dependency injection error we have in root.go at runtime, and then we can assert if this is the case.

Reply:
Merging and checking this in a follow-up